etc/kayobe/kolla/config/keystone/wsgi-keystone.conf

```diff
@@ -0,0 +1,63 @@
+{% raw %}
+{% set keystone_log_dir = '/var/log/kolla/keystone' %}
+{% set python_path = '/usr/lib/python2.7/site-packages' if kolla_install_type == 'binary' else '/var/lib/kolla/venv/lib/python2.7/site-packages' %}
+Listen {{ api_interface_address }}:{{ keystone_public_port }}
+Listen {{ api_interface_address }}:{{ keystone_admin_port }}
+
+TraceEnable off
+
+<VirtualHost *:{{ keystone_public_port }}>
+    WSGIDaemonProcess keystone-public processes={{ openstack_service_workers }} threads=1 user=keystone group=keystone display-name=%{GROUP} python-path={{ python_path }}
+    WSGIProcessGroup keystone-public
+    WSGIScriptAlias / /var/www/cgi-bin/keystone/main
+    WSGIApplicationGroup %{GLOBAL}
+    WSGIPassAuthorization On
+    <IfVersion >= 2.4>
+      ErrorLogFormat "%{cu}t %M"
+    </IfVersion>
+    ErrorLog "{{ keystone_log_dir }}/keystone-apache-public-error.log"
+    LogFormat "%{X-Forwarded-For}i %l %u %t \"%r\" %>s %b %D \"%{Referer}i\" \"%{User-Agent}i\"" logformat
+    CustomLog "{{ keystone_log_dir }}/keystone-apache-public-access.log" logformat
+{% endraw %}
+    # OpenID Connect configuration for EGI Check-in AAI
+    OIDCResponseType "code"
+    OIDCClaimPrefix "OIDC-"
+    OIDCClaimDelimiter ;
+    OIDCScope "openid"
+    OIDCProviderMetadataURL https://aai-dev.egi.eu/oidc/.well-known/openid-configuration
+    OIDCClientID {{ secrets_egi_client_id }}
```

Comment:
nit: Is this one actually a secret?

Reply:
Not necessarily secret, but arguably sensitive.